src/PayPal/Api/Subscription/BillingInfo/LastPayment/Amount.php

```diff
@@ -0,0 +1,15 @@
+<?php
+declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\PayPal\Api\Subscription\BillingInfo\LastPayment;
+
+use Swag\PayPal\PayPal\Api\Subscription\BillingInfo\OutstandingBalance;
+
+class Amount extends OutstandingBalance
```

Comment:
extend from https://github.com/shopwareLabs/SwagPayPal/blob/master/src/PayPal/Api/Common/Value.php

Reply:
changed to money